_includes/head.html

```diff
@@ -29,4 +29,15 @@
       height: 100%;
     }
   </style>
-<![endif]-->
\ No newline at end of file
+<![endif]-->
+
+{% if site.head_scripts %}
+  {% for script in site.head_scripts %}
+    {% if script contains "://" %}
```

Comment:
Maybe just "//" to allow for protocol-relative URLs (below too): https://stackoverflow.com/questions/4831741/can-i-change-all-my-http-links-to-just/27999789

Reply:
Protocol less URLs are now considered an anti-pattern. It's preferable to use `https` when available.

https://www.paulirish.com/2010/the-protocol-relative-url/

Reply:
I know, that's exactly what I answered on the very same SO thread :) Just thought you'd might want to support it, but I suppose there's really no need, any CDN today would have HTTPS...

Reply:
Exactly. And I'm not sure if `//` might filter out something it shouldn't be. I doubt anyone would have a path with that, but seems possible.

Reply:
Yeah I had the same concern, theoretically `https://github.com/mmistakes//minimal-mistakes` works too. I think even `https://github.com/mmistakes://minimal-mistakes` could work but since Liquid doesn't contain any `startswith` method that I can tell probably not worth the trouble.

Reply:
Sure. Keeping things consistent with the rest of the "external URL" checks in the theme. You may be over thinking things a tad, simple is better.

Reply:
Overthinking is my middle name 😅